sdcm/tester.py

```diff
@@ -405,7 +405,7 @@ def send_argus_heartbeat(client: ArgusSCTClient, stop_signal: threading.Event):
         thread_stop_signal = threading.Event()
         thread = threading.Thread(name="argus-heartbeat",
                                   target=send_argus_heartbeat,
-                                  kwargs={"connection": self.test_config.argus_client(), "stop_signal": thread_stop_signal})
+                                  kwargs={"client": self.test_config.argus_client(), "stop_signal": thread_stop_signal})
```

Comment:
Looks like this second commit could be squashed into the first one.

Reply:
Done